pkg/controllers/validationController.go

```diff
@@ -29,35 +29,33 @@ func NewValidationController() *ValidationController {
 func (c *ValidationController) Validate(w http.ResponseWriter, req *http.Request) {
 	writer := responseWriter.New(w)
 
+	err := headerValidation(req)
```

Comment:
why move this to the top of the controller? what will happen if a panic is thrown here? (before defer)